imagetagger/imagetagger/settings_base.py

```diff
@@ -13,16 +13,14 @@
 import os
 from django.contrib.messages import constants as messages
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-
+FS_URL = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+TMP_FS_URL = 'temp://imagetagger'
```

Comment:
We would like to use `settings_base.py` mostly for settings which make the application work with django and then use `settings.py` for more user-specific settings.
Since these are deployment-specific settings I would like to put them in `settings.py`

Reply:
I put them here in ```settings_base.py```, because ```IMAGE_PATH``` and ```TMP_IMAGE_PATH``` depend on them. Would it make sense for you to create a follow up pull request to move all of ```IMAGE_PATH```, ```TMP_IMAGE_PATH```, ```TOOLS_PATH```, ```TOOLS_ENABLED```, ```TOOL_UPLOAD_NOTICE``` and ```IMAGE_EXTENSION``` etc. to ```settings.py```?

Reply:
Never mind, just saw you did that already in the patches you attached.